bfvmm/include/hve/arch/intel_x64/vmexit/wrmsr.h

```diff
@@ -275,6 +276,8 @@ class EXPORT_HVE wrmsr_handler
     /// @endcond
 };
 
+using wrmsr_handler_delegate_t = wrmsr_handler::handler_delegate_t;
```

Comment:
Why are there specific types for each handler? I thought all handlers would have the same type with this change. Or maybe that's yet to come?

Reply:
This will be removed once I complete all of the APIs. This was added for Boxy in the mean time.

Reply:
:+1: